src/EntityDb.Abstractions/ValueObjects/TimeStamp.cs

```diff
@@ -24,4 +25,15 @@ namespace EntityDb.Abstractions.ValueObjects;
     /// <returns>A <see cref="TimeStamp"/> rounded down to the nearest millisecond.</returns>
     public TimeStamp WithMillisecondPrecision() =>
         new(Value - TimeSpan.FromTicks(Value.Ticks % TimeSpan.TicksPerMillisecond));
+
+    /// <summary>
+    ///     Converts the value of the current <see cref="TimeStamp"/> object to
+    ///     its equivalent string representation using the formatting
+    ///     conventions of the current culture.
+    /// </summary>
+    /// <returns>
+    ///     A string representation of the value of the current
+    ///     <see cref="TimeStamp"/> object.
+    /// </returns>
+    public override string? ToString() => Value.ToString(CultureInfo.InvariantCulture);

```

Comment:
One thing did pop out at me

`DateTime.ToString()` uses `CultureInfo.CurrentCulture`, I think it should be consistent

```suggestion
    public override string? ToString() => Value.ToString(CultureInfo.CurrentCulture);
```